pegasus/rake/generate_pdfs.rake

```diff
@@ -18,23 +19,57 @@ def pdf_conversions_for_files(file_pattern, url_extension)
   end
 end
 
+def pdf_conversions_for_state_pages(state_codes)
+  [].tap do |state_conversion_infos|
+    state_codes.each do |state_code|
+      file_path_without_makepdf_extension = sites_v3_dir('code.org/public/advocacy/state-facts/') + state_code
+      file_path_without_extension = file_path_without_makepdf_extension
+      url_path_from_public_without_extension = file_path_without_extension.match(/public(.*)/)[1]
+      url_path = url_path_from_public_without_extension + '?pdf_version=true'
+      state_conversion_infos << PDFConversionInfo.new(url_path, [file_path_without_makepdf_extension], file_path_without_extension + '.pdf')
+    end
+  end
+end
+
 base_url = ENV['base_url']
 
+def generate_pdf_files(base_url, pdf_conversion_info, fetchfile_for_pdf)
```

Comment:
how many pdf files does this generate? if it's always 1 I would prefer to call this `generate_pdf_file` or maybe even `generate_pdf` if the 'file' part can be assumed